src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -2,8 +2,258 @@
+            <Setter Property="Foreground" Value="{DynamicResource {x:Static shell:VsBrushes.InactiveCaptionTextKey}}" />
+        </Style>
+
+        <Style x:Key="OverlayGridStyle" TargetType="Grid">
+            <Setter Property="Background" Value="{DynamicResource {x:Static shell:VsBrushes.BrandedUIBackgroundKey}}" />
+            <Setter Property="Visibility" Value="Collapsed" />
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding CurrentIssue, TargetNullValue=1}" Value="1">
+                    <Setter Property="Grid.Visibility" Value="Visible"/>
+                </DataTrigger>
+            </Style.Triggers>
+        </Style>
+
+        <Style x:Key="DataFlowGridStyle" TargetType="Grid">
+            <Setter Property="Visibility" Value="Visible" />
+            <Style.Triggers>
+                <DataTrigger Binding="{Binding CurrentIssue, TargetNullValue=1}" Value="1">
+                    <Setter Property="Grid.Visibility" Value="Collapsed"/>
+                </DataTrigger>
+            </Style.Triggers>
+        </Style>
+
+        <!-- Data grid items styles -->
+
+        <Style x:Key="IssueDescriptionDockPanelStyle" TargetType="DockPanel">
+            <Setter Property="Margin" Value="5"/>
+        </Style>
+
+        <Style x:Key="StepNumberRectangleStyle" TargetType="Rectangle" BasedOn="{StaticResource BaseShapeStyle}">
+            <Setter Property="RadiusX" Value="1"/>
+            <Setter Property="RadiusY" Value="1"/>
+            <Setter Property="StrokeThickness" Value="1"/>
+        </Style>
+
+        <Style x:Key="LocationsListViewItemStyle" TargetType="ListViewItem">
+            <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+            <Setter Property="Padding" Value="3"/>
+            <Setter Property="Template">
+                <Setter.Value>
+                    <ControlTemplate TargetType="{x:Type ListViewItem}">
+                        <Border BorderThickness="0" 
+                                Background="{TemplateBinding Background}" 
+                                Padding="{TemplateBinding Padding}" 
+                                SnapsToDevicePixels="true">
+                            <ContentPresenter 
+                                HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}" 
+                                SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" 
+                                VerticalAlignment="{TemplateBinding VerticalContentAlignment}"/>
+                        </Border>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+            <Style.Triggers>
+                <DataTrigger x:Name="FileNameListItemStyleTrigger" Binding="{Binding Path=Location}" Value="{x:Null}">
+                    <Setter Property="IsEnabled" Value="False"/>
+                    <Setter Property="IsSelected" Value="False"/>
```

Comment:
How does this interact with the `IsSelected` trigger below? Does it means the user can't select those items in the list?

Reply:
Yep, exactly. The "file" list items are non-selectable.